### ISSUE TITLE
Ch5, Parthenia, Kiryl has party chat line when he's incapacitated 

### ISSUE THREAD

**Issue by ugoplatamia** [concreted/dqiv-nds-en-script-patcher#9]
When entering Parthenia the first time, when using party chat, Kiryl has a line even though he is not a useable party member by that point.

![2022051712452300-0A347E222F9DAE29C9514AB6C262481F](https://user-images.githubusercontent.com/19904207/168897315-7bc657c2-40c7-4b72-8b1f-481e64498950.jpg)

Also uploaded a save file, just enter Parthenia and use party chat to see the erroneous line. 
[Dragon Quest IV - Michibikareshi Monotachi (Party Chat).zip](https://github.com/concreted/dqiv-nds-en-script-patcher/files/8711145/Dragon.Quest.IV.-.Michibikareshi.Monotachi.Party.Chat.zip)

**Comment by concreted** [comment]
This tool only works off of existing lines in the party chat files, so this is likely an issue with the party chat files themselves, not the tool. Did you apply the patch with the files from the Android app?

**Comment by ugoplatamia** [comment]
Yeah, took the ENG .mpt files and did everything. I guess to really narrow this out, is to see if it occurs in the unpatched JP original.

**Comment by concreted** [comment]
This might not occur in the unpatched JP ROM because it would normally use the original Japanese set of party chat files. I'm thinking this could happen if the Android English party chat files include this line. 

To test we could try placing the raw party chat files (without running script patching) from the Android app into the JP ROM, and see if this line still shows up. I haven't had time to revisit this project recently but if I have some time I'll try reproducing, otherwise if you want to give that a try would be very helpful.

**Comment by scbroede** [comment]
this is actually just a mistake in the translation in the mobile version. it's supposed to be borya speaking, not kiryl. here's the same line in japanese (ブライ is borya's japanese name):
![image](https://github.com/concreted/dqiv-nds-en-script-patcher/assets/32247405/717cacdb-ff9c-4ec3-b5c6-82c70ccbf2e7)
